testcases.py

```diff
@@ -379,6 +387,42 @@ def check(self):
       return False
     return self._check_files()
 
+
+class TestCaseHandshakeLoss(TestCase):
+  @staticmethod
+  def name():
+    return "handshakeloss"
+
+  @staticmethod
+  def testname():
+    return "multiconnect"
+
+  @staticmethod
+  def abbreviation():
+    return "L"
+
+  @staticmethod
+  def timeout() -> int:
+    return 300
+
+  @staticmethod
+  def scenario() -> str:
+    """ Scenario for the ns3 simulator """
+    return "drop-rate --delay=15ms --bandwidth=10Mbps --queue=25 --rate_to_server=30 --rate_to_client=30"
+
+  def get_paths(self):
+    for _ in range(100):
```

Comment:
Can you set the 100 as a class constant? That will make it easier to play around with the test case.

e.g. like here:
https://github.com/marten-seemann/quic-interop-runner/blob/2d59be0aee61475829c031e9e606906dc375cf8f/testcases.py#L363-L364

Reply:
yup, done.